08-collab.md

```diff
@@ -97,3 +97,12 @@ Fast-forward
  1 file changed, 1 insertion(+)
  create mode 100644 pluto.txt
 ~~~
+
+> ## Review changes {.challenge}
+>
+> The Owner makes some changes in the repository without giving any information
```

Comment:
I'd prefer to use 'push commits' instead of 'make changes' to make clear that we're talking about new commits in the repo and not just uncommited changes in the working directory.